OMEdit/OMEditLIB/Element/ElementProperties.cpp

```diff
@@ -1297,7 +1297,7 @@ void ElementParameters::applyFinalStartFixedAndDisplayUnitModifiers(Parameter *p
     /* Ticket #2531
      * Check if parameter is marked final.
      */
-    if ((checkFinal && pModifier->isFinal()) || (!isElementModification && pModifier->isRedeclare() && !pModifier->isReplaceable())) {
+    if (checkFinal && pModifier->isFinal()) {
```

Comment:
Is this change intentional? I believe this is by mistake since these conditions are needed.

Reply:
Yes, with this condition you can`t change redeclare component after you set it once and press OK. This parameter is disappear from parameter list. I will return this condion and open another issue, because this is another problem.